docs/data-sources/vpcep_endpoints.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "VPC Endpoint (VPCEP)"
+---
+
+# flexibleengine\_vpcep\_endpoints
+
+Use this data source to get VPC endpoints.
+
+## Example Usage
+
+```hcl
+data "flexibleengine_vpcep_endpoints" "all_services" {
```

Comment:
maybe "all_endpoints" will be more clearly

Reply:
done